javascript/javascript3/week2/visual.js

```diff
@@ -0,0 +1,63 @@
+//target move one by one
+async function translateOneByOne() {
+    await new Promise((resolve) => {
+      resolve(
+        moveElement(document.querySelector("ul.marks li:nth-child(1)"), {
+          x: 20,
+          y: 300,
+        })
+      );
+      console.log("first");
+    });
+    await new Promise((resolve) => {
+      resolve(
+        moveElement(document.querySelector("ul.marks li:nth-child(2)"), {
```

Comment:
You already have `redBox, blueBox, greenBox` as declared variables in `move-element.js` file. You don't need to use the query selector each time. Just use the existing variables